.github/workflows/build.yml

```diff
@@ -74,3 +74,10 @@ jobs:
         with:
           name: packages
           path: src/**/*.nupkg
+
+      - name: Archive test results
+        uses: actions/upload-artifact@v3
+
+        with:
+          name: test-results
+          path: tests/**/TestResults/*.trx
```

Comment:
will this run if the tests step fails still? And how will these results upload for different 'runs', will they overlap?

Reply:
No idea 😅

Reply:
Somehow, it seems the results are merged - 
<img width="928" alt="image" src="https://user-images.githubusercontent.com/5427083/202743093-2819d76d-5bb8-4aaf-b946-b2c9ff703261.png">

There's no failing results in this case because they're timeouting instead of failing, I guess